src/mozanalysis/metrics.py

```diff
@@ -140,17 +186,24 @@ def build_query(
         time_limits: TimeLimits,
         experiment_slug: str,
         from_expr_dataset: str | None = None,
-        analysis_basis: str = AnalysisBasis.ENROLLMENTS,
+        analysis_basis: AnalysisBasis = AnalysisBasis.ENROLLMENTS,
+        experimental_unit: AnalysisUnit = AnalysisUnit.CLIENT,
```

Comment:
Should we use `analysis_unit` here, or is `experimental_unit` more correct in this case?

Reply:
No, I think that's old and needs to be updated. `analysis_unit` should be right.